share/completions/xxd.fish

```diff
@@ -0,0 +1,20 @@
+set -l xxd_exclusive_args b e i ps r
+
+complete -c xxd -s a -d 'Toggle autoskip'
+complete -c xxd -s b -n "__fish_complete_exclusives $xxd_exclusive_args" -d 'Binary digit dump'
```

Comment:
What is `__fish_complete_exclusives`?

Reply:
That's a custom completion helper that I forgot was custom, oops 😅 I'll inline it in the file.